src/meshes/t8code_mesh.jl

```diff
@@ -1077,20 +1074,15 @@ function fill_mesh_info!(mesh::T8codeMesh, interfaces, mortars, boundaries,
                                 push!(visited_global_mortar_ids, global_mortar_id)
                                 global_mortar_id_to_local[global_mortar_id] = local_mpi_mortar_id
 
-                                mpi_mesh_info.mpi_mortars.local_neighbor_ids[local_mpi_mortar_id] = [
-                                    current_index + 1,
-                                ]
-                                mpi_mesh_info.mpi_mortars.local_neighbor_positions[local_mpi_mortar_id] = [
-                                    map_iface_to_ichild_to_position[iface + 1][t8_element_child_id(eclass_scheme, element) + 1],
-                                ]
+                                mpi_mesh_info.mpi_mortars.local_neighbor_ids[local_mpi_mortar_id] = [current_index +
+                                                                                                     1]
+                                mpi_mesh_info.mpi_mortars.local_neighbor_positions[local_mpi_mortar_id] = [map_iface_to_ichild_to_position[iface + 1][t8_element_child_id(eclass_scheme, element) + 1]]
```

Comment:
This lines gets really long.

Reply:
Did you check if a line break behind e.g. the `=` gets removed from the formater?

Reply:
Yes, after the `=`, it does get removed. I've pushed an alternative in [f393d48](https://github.com/trixi-framework/Trixi.jl/pull/1842/commits/f393d484dd7b0f2d08372e5009d0f67e861aa2da) with a line break after the first closing bracket.

Reply:
Oh, never mind. This does not work.

Reply:
So none of the tried options work?

Reply:
Unfortunately yes. We could, however, give shorter names to `map_iface_to_ichild_to_position[iface + 1]` and `t8_element_child_id(eclass_scheme, element) + 1` before.

Reply:
Maybe we should just push this trough to get the other PRs going